playbooks/robusta_playbooks/alerts_integration.py

```diff
@@ -84,6 +85,39 @@ def silence_alert(alert: PrometheusKubernetesAlert, params: SilenceAlertParams):
     alert.stop_processing = True
 
 
+class StatusSilencerParams(ActionParams):
+    """
+    :var include: If available, will stop processing unless the pod status is in the include list
+    :var exclude: If available, will stop processing if the pod status is in the exclude list
+
+    :example include: ["Pending"]
+    :example exclude: ["Evicted"]
+    """
+
+    include: Optional[List[str]]
+    exclude: Optional[List[str]]
+
+
+@action
+def pod_status_silencer(event: PodEvent, params: StatusSilencerParams):
+    """
+    Stop execution based on pod statuses.
+    """
+    pod = event.get_pod()
+    if not pod:
+        logging.info("Cannot run pod_status_silencer with no pod. skipping")
+        return
+
+    if params.include:  # Stop unless pod status in include list
+        if pod.status.phase not in params.include:
```

Comment:
are you sure you want to use phase for the status's
Its very high level. 
Is that what you want to filter or do you want to filter out more specific status like crashloop, oomkill, imagepull?
because allot of them will be contained in the the phase running like crashloop for example:

`containerStatuses:
  - containerID: containerd://656cd4aca1ea7cba7821dfe572908308709f3570898fd35518fb085fb026651c
    image: docker.io/library/busybox:latest
    lastState:
 ...
    state:
      waiting:
        message: back-off 20s restarting failed container=crashpod pod=crashpod-64b874bbbc-8rqts_default(cb100c54-b174-4d87-b48d-1e54c1c375b4)
        reason: CrashLoopBackOff
  phase: Running
`

Reply:
The user asked for phase. Not sure if "Evicted" is a phase